js/gameConfig.js

```diff
@@ -24,6 +24,8 @@ const units = {
     storedEnergy: 'Wh'
 };
 
+const gridStrength = new GridStrength({name:'GridStrength', title: 'Grid Strength', maxXp: 100, effects: [{effectType: EffectType.GridStrength, baseValue: 1}]});
```

Comment:
okay ... yeah. 🤯 GridStrength hat einfach selbst nen Effect der GridStrength angibt. Bin mir nicht sicher ob das genial oder wahnsinnig ist.
Also da ja eigentlich die Ansage war; Level von GridStrength = GridStrength würde ich das wohl auch bei dieser simplen Logik lassen, ohne extra Effect Layer

Reply:
so funktionieren tasks jetzt halt... ohne effectType kein effect. 💯

Reply:
Ja aber GridStrength ist halt auch kein Effect am Ende des Tages. Das is nen Level von nem Task. Nen Task den man zwar nicht deaktivieren kann, aber halt progressed.
Hast ja sogar die Methode `GridStrength.prototype.getGridStrength` die das Level verwendet.